tasks/translation-ollama/main.py

```diff
@@ -0,0 +1,81 @@
+from pathlib import Path
+
+import logging
+import zipfile
+
+import fitz
+import ollama
+
+from model import launchOllamaServer, pullModel, LLM
+
+from coretex import currentTaskRun, CustomDataset
+
+
+def readPDF(filePath: Path) -> list[str]:
+    pagesText: list[str] = []
+    with fitz.open(filePath) as doc:
+        for page in doc:
+            paragraphs = page.get_text().split("\n")
+            pagesText.extend(paragraphs)
+
+    return pagesText
+
+
+def loadCorpus(dataset: CustomDataset) -> list[list[str]]:
+    corpus: list[list[str]] = []
+    for sample in dataset.samples:
+        sample.unzip()
+        for pdfPath in sample.path.glob("*.pdf"):
```

Comment:
Use `Path.rglob` instead of `Path.glob`. `Path.rglob` will recursively search all nested directories as well, not just the top level directory.